docs/uptime-guide/install.asciidoc

```diff
@@ -51,7 +51,7 @@ Uptime requires the setup of monitors in Heartbeat.
 These monitors provide the data you'll be visualizing in the {kibana-ref}/xpack-uptime.html[Uptime app].
 
 For instructions on installing and configuring Heartbeat, see the *Setup Instructions* in {kib}.
-Additional information is available in {heartbeat-ref}/heartbeat-configuration.html[Configure Heartbeat].
+Additional information is available in {heartbeat-ref}/configuring-howto-heartbeat.html[Configure Heartbeat].
```

Comment:
@benskelker Would you prefer that this link point to the quick start guide instead of the config topic?

Reply:
Actually, I was having some problems with the CI for this PR, so I closed it. Respond here instead please: https://github.com/elastic/kibana/pull/70380/files#diff-772dc80e57e7a6fafe1d9ebf3359b43eR54

Reply:
@dedemorton thanks, but you're asking the wrong writer. Maybe @bmorelli25?

Reply:
@benskelker haha. sorry. Brain glitch.